### DISCUSSION TITLE
Transcription issue with audio files longer than 10 minutes when CUDA is enabled - Linux Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by jgoer** [openai/whisper#2211] [Q&A] [answered]
Hello,

I am encountering an issue with Whisper.cpp. When I try to transcribe a WAV file longer than 10 minutes (around 40 MB), Whisper returns an infinite loop containing "[BLANK_AUDIO]" or sometimes "– Subtitling: Le Crayon d'oreille".

I am using version v1.6.2 of Whisper.cpp. My OS is Ubuntu 20.04, NVIDIA Drivers 535.171.04 and CUDA version is 12.3. My GPU is an NVIDIA A40. I have tested with the models ggml-large-v2.bin and ggml-large-v3.bin, but the problem remains the same. When I disable the GPU version with the flag --no-gpu, the transcription proceeds without any issue.

Has anyone else encountered this problem?

Thanks in advance.

**Comment by glangford** [comment, marked as answer]
Better to ask over at the repo for whisper.cpp

https://github.com/ggerganov/whisper.cpp
